actors/market/src/lib.rs

```diff
@@ -712,7 +817,7 @@ impl Actor {
                 }
 
                 let mut state: DealState = st
-                    .find_deal_state(rt.store(), id)?
+                    .find_deal_state(rt.store(), *id)?
```

Comment:
Then loading the deal states, over and over. AHHHHHHHHHHHHHHHH.

(ok, maybe file a bug and move on)?

Reply:
https://github.com/filecoin-project/builtin-actors/issues/1498